.changelog/4490529bdb8044068d99216cba424862.json

```diff
@@ -0,0 +1,8 @@
+{
+    "id": "4490529b-db80-4406-8d99-216cba424862",
+    "type": "feature",
+    "description": "Remove deprecated `nimble` service.",
+    "modules": [
+        "."
```

Comment:
nit: shouldn't this be scoped to `nimble` instead of `.`?

Reply:
So technically the `nimble` module is being deleted -- I'm not sure if our tooling is written to handle that as an edge case. I suspect it will try to go update `service/nimble/CHANGELOG.md` to include this new entry, but that file wouldn't exist anymore since we deleted it, so the release would probably explode.

The other problem we have (recently noted in #2819, not directly although I think this is another case of that) is that a changelog like this bumps the root SDK module for basically no reason (no actual change) which by proxy bumps every service client module for equally no reason.

TL;DR not sure what to do here. We should honestly probably just not have a changelog for these types of deprecations for now.

Reply:
involved modules are generated by changelog cmd according to which commit we use, here I passed the commit including both model and service api removal